src/plugins/intel_gpu/src/graph/scatter_update.cpp

```diff
@@ -37,8 +37,8 @@ layout scatter_update_inst::calc_output_layout(scatter_update_node const& node)
 
     const int32_t axis = desc->axis;
     const size_t indices_size = node.input(1).get_output_layout().count();
-    const size_t input_number_of_dims = node.input(0).get_output_layout().get_rank();
-    const size_t updates_number_of_dims = node.input(2).get_output_layout().get_rank();
+    const size_t input_number_of_dims = node.input(0).get_output_layout().get_tensor().sizes().size();
```

Comment:
How about replace this commit of scatter_update fix to https://github.com/ahnyoung-paul/openvino/commit/b06e02b9158c493e09f1416bfc5f477efdc956a2 ?